src/services/focusBlur.js

```diff
@@ -34,7 +34,10 @@ Controller.open(function(_) {
     }
     function blur() { // not directly in the textarea blur handler so as to be
       cursor.hide().parent.blur(); // synchronous with/in the same frame as
-      ctrlr.root.jQ.removeClass('mq-focused'); // clearing/blurring selection
+      // Sometimes ctrlr.root can be undefined due to race condition
```

Comment:
I was being a bit vague 😄 

Basically I think in certain circumstances we call some method on mathquill that removes the ctrlr properties, before the deferred [blurTimeout hander](https://github.com/Learnosity/mathquill/blob/e39f5ac9ca7724a352c6f6f0f5f9cab47d511674/src/services/focusBlur.js#L23-L27) has a chance to execute, causing this exception.

I left some more detailed debugging info in a [JIRA comment](https://learnosity.atlassian.net/browse/LRN-12436?focusedCommentId=48302&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-48302).